sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -230,7 +239,11 @@ object BasicWriteJobStatsTracker {
       NUM_FILES_KEY -> SQLMetrics.createMetric(sparkContext, "number of written files"),
       NUM_OUTPUT_BYTES_KEY -> SQLMetrics.createSizeMetric(sparkContext, "written output"),
       NUM_OUTPUT_ROWS_KEY -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
-      NUM_PARTS_KEY -> SQLMetrics.createMetric(sparkContext, "number of dynamic part")
+      NUM_PARTS_KEY -> SQLMetrics.createMetric(sparkContext, "number of dynamic part"),
+      DURATION_OF_TASK_COMMIT ->
+        SQLMetrics.createTimingMetric(sparkContext, "duration of task commit"),
+      DURATION_JOB_COMMIT->
+        SQLMetrics.createTimingMetric(sparkContext, "duration of committing the job")
```

Comment:
Maybe, `committing the job` -> `job commit`?

Reply:
Done